### PR TITLE
Updated cursed examples

### PR DESCRIPTION
**Pull request by kilowatt-**

### CODE REVIEW

File: examples/undefined.js [hunk 1]
```diff
@@ -5,24 +5,26 @@ function get(id) {
 }
```
Comment (curtiskoo) [inline examples/undefined.js:5]: did you want to remove `get()` as well?

File: examples/fizzybuzzy.js [hunk 2]
```diff
@@ -8,7 +8,8 @@
  * In this modified version, we take in an arbitrary list of numbers as input
  * and return a list of string / number results instead of printing
  */
-function fizzBuzzList(inputs) {
+const inputs = [...Array(20).keys()];
+
```
Comment (curtiskoo) [inline examples/fizzybuzzy.js:12]: the indentation on this file is cursed, did you mean to do that?

**Review by curtiskoo — APPROVED** [review]

File: examples/fizzybuzzy.js (hunk 2 above)
Comment (kilowatt-) [reply examples/fizzybuzzy.js:12]: nope let me fix it LOL

File: examples/undefined.js (hunk 1 above)
Comment (kilowatt-) [reply examples/undefined.js:5]: good catch

File: examples/fizzybuzzy.js (hunk 2 above)
Comment (jlu5) [reply examples/fizzybuzzy.js:12]: I'm not sure we'll actually be able to infer the type of this, maybe just hardcoding a list of ints would be better.

Comment (kilowatt-) [reply examples/fizzybuzzy.js:12]: fixed @jlu5

**Review by jlu5 — APPROVED** [review]

**Pull request merged**